SpatialGDK/Source/SpatialGDKTests/Examples/SpatialGDKExampleTest.cpp

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "HAL/PlatformFilemanager.h"
+#include "Misc/ScopeTryLock.h"
+
+#define EXAMPLE_SIMPLE_TEST(TestName) \
+	GDK_TEST(SpatialGDKExamples, SimpleExamples, TestName)
+
+#define EXAMPLE_COMPLEX_TEST(TestName) \
+	GDK_COMPLEX_TEST(SpatialGDKExamples, ComplexExamples, TestName)
+
+DECLARE_LOG_CATEGORY_EXTERN(LogSpatialGDKExamples, Log, All);
+DEFINE_LOG_CATEGORY(LogSpatialGDKExamples);
+
+// 1. Latent command example
+namespace
+{
+const double MAX_WAIT_TIME_FOR_BACKGROUND_COMPUTATION = 2.0;
+const double MIN_WAIT_TIME_FOR_BACKGROUND_COMPUTATION = 1.0;
+const double COMPUTATION_DURATION = 1.0;
+
+struct ComputationResult
+{
+	FCriticalSection Mutex;
+	int Value = 0;
+};
+
+TUniquePtr<ComputationResult> ResultPtr = nullptr;
```

Comment:
This doesn't really give us anything if we still have to manually new and delete the object.